platzky/feature_flags.py

```
@@ -0,0 +1,268 @@
+        upper_name = name.upper()
+        flag_cls = self._alias_to_flag.get(upper_name)
+        if flag_cls is not None:
+            return self._values[flag_cls]
+
+        # Try unregistered
+        if name in self._unregistered:
+            return self._unregistered[name]
+        if upper_name in self._unregistered:
+            return self._unregistered[upper_name]
+
+        raise AttributeError(f"FeatureFlags has no flag '{name}'")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)